gallery/src/styles/NxList/NxListDefaultExample.html

```diff
@@ -6,33 +6,21 @@
     distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
 
 -->
-<div class="nx-list">
-  <h4 class="nx-list__title">
-    List title
-  </h4>
-  <ul>
-    <li class="nx-list__item">
-      item 1
-    </li>
-    <li class="nx-list__item">
-      Item 2
-    </li>
-    <li class="nx-list__item">
-      item 3 - this list item should be truncated at the right end edge. youtube weathered network network systemic
-      systema claymore mine voodoo god garage monofilament realism order-flow corporation car footage vinyl.
-    </li>
-  </ul>
-</div>
-
+<h3 class="nx-h3">
+  Basic lists
+</h3>
 <ul class="nx-list">
   <li class="nx-list__item">
-    Item 1 - nx-list can also be applied directly to a &lt;ul&gt; or &lt;ol&gt;.
+    <span class="nx-list__text">Item 1</span>
```

Comment:
With some tweaks to the `.nx-list__item` styles, I think it should be possible to make `.nx-list__text` optional for simple list items, or maybe even to get rid of it entirely.  The first bare text node within the grid container will occupy the first (empty?) grid cell, which is conveniently exactly where we'd want it.

Reply:
> With some tweaks to the `.nx-list__item` styles, I think it should be possible to make `.nx-list__text` optional for simple list items, or maybe even to get rid of it entirely. The first bare text node within the grid container will occupy the first (empty?) grid cell, which is conveniently exactly where we'd want it.

We go to lots of effort throughout the RSC to make sure that we don't have bare text nodes, why change that here?

Reply:
> We go to lots of effort throughout the RSC to make sure that we don't have bare text nodes, why change that here?

I was just thinking about the case where it's a "simple" nx-list that just has the main text per row and none of the other elements, and whether we'd want to streamline that case by making the `.nx-list__text` wrapper optional in that case.  I guess it depends on how common we see that case being.  Maybe it's not that common.

Reply:
> > We go to lots of effort throughout the RSC to make sure that we don't have bare text nodes, why change that here?
> 
> I was just thinking about the case where it's a "simple" nx-list that just has the main text per row and none of the other elements, and whether we'd want to streamline that case by making the `.nx-list__text` wrapper optional in that case. I guess it depends on how common we see that case being. Maybe it's not that common.

Especially now that we're using CSS Grid I would really prefer not to create any confusion around when the `span` should be used.